rdr_service/api/site_hierarchy_api.py

```diff
@@ -0,0 +1,14 @@
+from flask import request
+
+from rdr_service.api.base_api import BaseApi
+from rdr_service.dao.site_hierarchy_dao import SiteHierarchyDao
+
+
+class SiteHierarchyApi(BaseApi):
+    def __init__(self):
+        super().__init__(SiteHierarchyDao())
+
+    def list(self, participant_id=None):
```

Comment:
Do we know who access for this should be restricted to? We'd want to use a decorator here to make sure any requests are authenticated. And maybe the role should be curation, but I'm not sure if the EHR ops team is using the same service account.

`BaseApi`'s `get` method may also need to be overridden just to add in a auth check there too.

Reply:
Makes sense. Yes, I think this will mainly be called by curation.

Reply:
@kenny-skaggs I added the `get` method in there,  but I think it's redundant since it calls the `list` method, and the list method is protected with `@app_util.auth_required([...])`.

Reply:
Yeah, looking at it closer you're right. I wasn't quite sure on how that fit together.